docs/_data/prague-2020-sessions.yaml

```
@@ -0,0 +1,627 @@
+  slug: a-journey-to-pattern-languages-andrew-haynes
+  series: Write the Docs Prague
+  series_slug: a-journey-to-pattern-languages-andrew-haynes
+  year: 2020
+  speakers:
+  - name: Andrew Haynes
+    slug: andrew-haynes
+    twitter: '@CycleCoder'
+    website: 
+  abstract: '<p>For years, I have been sailing around the submerged iceberg of Pattern
+    Languages in my careers of software developer, technical author, information architect
+    and Scrum Master. I eventually landed on Christopher Alexander’s book, “A Pattern
+    Language” as a recommendation from my Design Thinking professor while studying
+    a master’s degree. Once I started to explore, I found connections everywhere.
+    Interestingly, pattern languages provide a unifying link between all my careers.
+
+    Pattern languages have had a transformative effect on numerous domains: built
+    architecture, information architecture, software design patterns, wikis, Scrum
+    and Agile to name a few. Yet for such an influential approach, pattern languages
+    remain surprisingly hidden.
+
+    I share my journey of discovery and rediscovery of language patterns. I will cover
+    a brief history of pattern languages and their impact on multiple domains. This
+    gives a context for my attempt to use pattern languages to link interests from
+    my various careers. This is primarily through the medium of a wiki which is a
+    direct descendent of the original explorations of pattern languages.
+
+    Finally, this is an opportunity to share resources that I have found useful on
+    my journey. The aim is to leave everyone with signposts to pattern languages in
+    multiple domains. Hopefully everyone will leave with some stimulating ideas about
+    how pattern languages may impact their own career.</p>'
+- title: The Baseline -- Or Technical Writing for Non-Technical Readers
+  slug: the-baseline-or-technical-writing-for-non-technical-readers-paul-brown
+  series: Write the Docs Prague
+  series_slug: the-baseline-or-technical-writing-for-non-technical-readers-paul-brown
+  year: 2020
+  speakers:
+  - name: Paul Brown
+    slug: paul-brown
+    twitter: '@linux_spain'
+    website: https://quickfix.es
+  abstract: <p>Technical writers are not always required to write exclusively for
+    engineers. Sometimes they also have to write for their bosses, their bosses' bosses,
+    investors, buyers and end users. These readers may or may not come from technical
+    background. In this talk we will look at ways you can ensure your message gets
+    across.</p>
+- title: The Rocky Road to DocOps
+  slug: the-rocky-road-to-docops-chris-noonan
+  series: Write the Docs Prague
+  series_slug: the-rocky-road-to-docops-chris-noonan
+  year: 2020
+  speakers:
+  - name: Chris Noonan
+    slug: chris-noonan
+    twitter: 
+    website: https://www.linkedin.com/in/cnoonan/
+  abstract: "<p><strong>Abstract</strong><br />\nTechnology and process are invariably
```

```suggestion
  abstract: "<p>Technology and process are invariably
```
(We should actually do that in pretalx and re-import)